Package.swift

```diff
@@ -130,7 +130,11 @@ let package = Package(
         ),
         .target(
             name: "SmithyTestUtil",
-            dependencies: ["ClientRuntime", "SmithyHTTPAPI"]
+            dependencies: [
+                "ClientRuntime",
+                "SmithyHTTPAPI",
+                .product(name: "AwsCommonRuntimeKit", package: "aws-crt-swift"),
```

Comment:
SmithyTestUtil now needs this dependency on CRT (this was an oversight in the earlier TestInitializer PR.)